modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -215,6 +234,19 @@ const Renderer = ({
 				page={page}
 				pageIndex={pageIndex}
 			>
+				{!isAdminPortletNamespace && hasFieldRequired && (
```

Comment:
@georgel-pop-lr, here is just missing use the alphabetic order, like:
```
{hasFieldRequired && !isAdminPortletNamespace && ...
```
Almost there. 🙂

Reply:
Hi @natocesarrego,

Sorry but for me, this doesn't make much sense, let me explain to you why I think so.
What if you want for some reasons to mix "&&" or "||" in the condition and changing position could impact the result? 
What if you want to check a condition first that consumes fewer computations so you improve the speed of the code?
Anyway if you believe this change is needed, I will do it, but I just want to have my mind clear for future changes.

Regards.

Reply:
Hi @georgel-pop-lr, no problem to disagree, actually this is good because it gives me the chance to learn from you and vice-versa. 🙂 

I asked you to perform this change because, by default, we should order it using alphabetic order, Brian Chan has requested it to me during a pull request review. Although, if we need to change the alphabetic order due to a important reason (use case, code speed, etc) then we can change. On this specific code, is the order proposed by you necessary?

If not, do you mind to use the alphabetic order?

Thanks in advance.

Reply:
Hi @natocesarrego,

As I said in my previous message there is no problem to change it, I just wanted to understand the reason behind this change. Plus I waited to clarify the other point with the message for required fields, so I can push both of them at the same time.

Thanks.
Regards.